easybuild/easyconfigs/r/R/R-4.2.1-foss-2022a.eb

```diff
@@ -3535,6 +3535,18 @@ exts_list = [
     ('svglite', '2.1.1', {
         'checksums': ['48700169eec1b05dbee9e2bae000aa84c544617b018cb3ac431a128cfd8dac56'],
     }),
+    ('ECOSolveR', '0.5.5', {
+        'checksums': ['2594ed1602b2fe159cc9aff3475e9cba7c1927b496c3daeabc1c0d227943ecc7'],
+    }),
+    ('scs', '3.2.4', {
+        'checksums': ['c3f39874bf4532fa8c2f2e2c41533ba4fe20b61cf6dfc6314407dc981621298f'],
+    }),
+    ('osqp', '0.6.0.8', {
```

Comment:
https://github.com/osqp/osqp-r/blob/master/configure#L27

If a system `cmake` is found then it is used, if not it uses a custom `Makefile` to build `osqp`. They'll be a test fail report from me due to the OS `cmake` failing (`XZ_5.2` symbol issue, see #14991 and #5518).

Reply:
ok, i'll add CMake builddep